PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/link/mysql/MySQLConnectionUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.link.mysql;
+
+import com.mysql.cj.jdbc.ConnectionImpl;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.sql.*;
+
+public class MySQLConnectionUtils {
```

Comment:
isn't this enough?
MySQLConnectionUtils{
    public static Connection connect(xxx);
    public static void ping(Connection connection) throw XxxException{
        
    }
}